[2023-09-04T06:01:09.270+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2021-12-27T00:00:00+00:00 [queued]>
[2023-09-04T06:01:09.293+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2021-12-27T00:00:00+00:00 [queued]>
[2023-09-04T06:01:09.293+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:01:09.330+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): delete_data_from_table> on 2021-12-27 00:00:00+00:00
[2023-09-04T06:01:09.342+0000] {standard_task_runner.py:57} INFO - Started process 4662 to run task
[2023-09-04T06:01:09.354+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'delete_data_from_table', 'scheduled__2021-12-27T00:00:00+00:00', '--job-id', '152', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmp6g7ox78b']
[2023-09-04T06:01:09.363+0000] {standard_task_runner.py:85} INFO - Job 152: Subtask delete_data_from_table
[2023-09-04T06:01:09.490+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2021-12-27T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:01:09.683+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='delete_data_from_table' AIRFLOW_CTX_EXECUTION_DATE='2021-12-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-12-27T00:00:00+00:00'
[2023-09-04T06:01:09.688+0000] {sql.py:274} INFO - Executing: 
            delete from dag_runs where dt = '2021-12-27' and dag_id = 'dag_with_postgres_operator_v04';
        
[2023-09-04T06:01:09.713+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:01:09.736+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:01:09.756+0000] {sql.py:418} INFO - Running statement: 
            delete from dag_runs where dt = '2021-12-27' and dag_id = 'dag_with_postgres_operator_v04';
        , parameters: None
[2023-09-04T06:01:09.765+0000] {sql.py:427} INFO - Rows affected: 0
[2023-09-04T06:01:09.788+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=delete_data_from_table, execution_date=20211227T000000, start_date=20230904T060109, end_date=20230904T060109
[2023-09-04T06:01:09.849+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:01:09.897+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-09-04T06:03:59.013+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2021-12-27T00:00:00+00:00 [queued]>
[2023-09-04T06:03:59.069+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2021-12-27T00:00:00+00:00 [queued]>
[2023-09-04T06:03:59.070+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:03:59.143+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): delete_data_from_table> on 2021-12-27 00:00:00+00:00
[2023-09-04T06:03:59.160+0000] {standard_task_runner.py:57} INFO - Started process 5978 to run task
[2023-09-04T06:03:59.179+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'delete_data_from_table', 'scheduled__2021-12-27T00:00:00+00:00', '--job-id', '698', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmp1kmkwmac']
[2023-09-04T06:03:59.186+0000] {standard_task_runner.py:85} INFO - Job 698: Subtask delete_data_from_table
[2023-09-04T06:03:59.351+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2021-12-27T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:03:59.662+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mohghaff' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='delete_data_from_table' AIRFLOW_CTX_EXECUTION_DATE='2021-12-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-12-27T00:00:00+00:00'
[2023-09-04T06:03:59.666+0000] {sql.py:274} INFO - Executing: 
            delete from dag_runs where dt = '2021-12-27' and dag_id = 'dag_with_postgres_operator_v04';
        
[2023-09-04T06:03:59.690+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:03:59.720+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:03:59.755+0000] {sql.py:418} INFO - Running statement: 
            delete from dag_runs where dt = '2021-12-27' and dag_id = 'dag_with_postgres_operator_v04';
        , parameters: None
[2023-09-04T06:03:59.769+0000] {sql.py:427} INFO - Rows affected: 1
[2023-09-04T06:03:59.804+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=delete_data_from_table, execution_date=20211227T000000, start_date=20230904T060359, end_date=20230904T060359
[2023-09-04T06:03:59.864+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:03:59.985+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
